src/lib/charm/openstack/keystone_saml_mellon.py

```diff
@@ -275,6 +271,13 @@ class KeystoneSAMLMellonCharm(charms_openstack.charm.OpenStackCharm):
     # ownership.
     group = 'www-data'
 
+    @property
+    def restart_map(self):
+        _map = {}
+        for config in CONFIGS:
+            _map[config] = []
+        return _map
+
     def configuration_complete(self):
```

Comment:
``charms.openstack`` has grown built-in support for checking for presence of and updating status for required configuration options.  I have not tried it myself, but I would like to ask if you have checked if it would fit your use case?  I believe @gnuoy added it here: https://github.com/openstack/charms.openstack/commit/45ffbcd9a45e2164c8d42771cb6df7080da70faf

Reply:
This does not quite work here. configuration_complete and custom_assess_status_check are verifying not just config but also resources and that those resources are valid XML